sci-mathematics/rstudio/rstudio-1.4.1106.ebuild

```diff
@@ -0,0 +1,97 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake qmake-utils
+
+QT_VER=5.12.6
+QT_SLOT=5
+
+DESCRIPTION="IDE for the R language"
+HOMEPAGE="
+	https://rstudio.com
+	https://github.com/rstudio/rstudio/"
+SRC_URI="
+	https://github.com/rstudio/rstudio/archive/v${PV}.tar.gz -> ${P}.tar.gz
+	https://s3.amazonaws.com/rstudio-dictionaries/core-dictionaries.zip
+"
+
+LICENSE="AGPL-3"
+SLOT="0"
+KEYWORDS="~amd64"
+# IUSE="dedicated libressl server"
+
+# ADD?
+# libsqlite3-dev (this is the ubuntu thingy) → dev-db/postgres
+# libpq-dev → dev-db/sqlite
+# yaml-cpp something?
```

Comment:
eix yaml-cpp? :p

```suggestion
# yaml-cpp -> dev-cpp/yaml-cpp
```

Reply:
Hah, indeed. I did find it, but ended up compiling fine without it.

I decided to leave the note to myself in case some issue pops up later during testing.

Reply:
so far it doesn't seem like it's needed...